app/models/library_location.rb

```diff
@@ -11,82 +11,21 @@ def initialize(library, location)
     @location = location
   end
 
-  def pickup_libraries
-    case
-    when location_specific_pickup_library_config.present?
-      location_specific_pickup_libraries
-    when library_specific_pickup_library_config.present?
-      library_specific_pickup_libraries
-    else
-      pickup_libraries_for(config.pickup_libraries)
-    end
-  end
-
   def active_messages
     @active_messages ||= Message.where(library: [library, 'anywhere']).active
   end
 
   class << self
     def library_name_by_code(code)
-      all_libraries[code] || location_specific_library_name_by_code(code)
-    end
-
-    def location_specific_library_name_by_code(code)
-      pickup_libraries_for_location = Array(config.location_specific_pickup_libraries[code])
-      return unless pickup_libraries_for_location.one?
-
-      all_libraries[pickup_libraries_for_location.first]
-    end
-
-    def config
-      SULRequests::Application.config
+      all_libraries[code]&.label
     end
 
     def all_libraries
-      config.libraries
+      Settings.libraries
     end
 
     def pageable_libraries
-      all_libraries.select { |k, _| config.pageable_libraries.include? k }
+      all_libraries.map.select { |_, v| v.pageable }.to_h.stringify_keys
```

Comment:
Hmm, interesting. What is the `.map` adding here?

Reply:
I couldn't get the RailsConfig `select` to work 🤷‍♂️

Reply:
(And.. you might ask `why not #to_h?`... `map` preserves the railsconfig objects further down the tree, and `to_h` turns everything in the tree into ordinary hashes)

Reply:
@cbeer I ... don't think I understand why that'd be the case at all, but if it works, great. Maybe toss these in a comment above this line as an act of kindness to posterity?

Reply:
Sure.

```
Settings.library.select # => nil
Settings.library.to_h # => { default: { ... }, ... }
Settings.library.map.to_h # { default: #<Config::Options  ... > }
```

Why it works that way, 🤷